docs/src/physics/coriolis_forces.md

```diff
@@ -61,4 +61,5 @@ the locally vertical and the locally horizontal components of the rotation vecto
     \boldsymbol{f} = \left[ 2\Omega\cos\varphi_0 \left( 1 -  \frac{z}{R} \right) + \gamma y \right] \boldsymbol{\hat y}
            + \left[ 2\Omega\sin\varphi_0 \left( 1 + 2\frac{z}{R} \right) + \beta  y \right] \boldsymbol{\hat z} \, ,
 ```
-where ``\beta = 2 \Omega \cos \varphi_0 / R`` and ``\gamma = -4 \Omega \varphi_0 / R``.
+as can be found in [DellarJFM2011](@cite) where 
```

Comment:
"in Dellar et al. 2011" reads as you are referring to Bill's internal organs :)

perhaps better is "in the work by Dellar..."?

[I know that this is a common way to phrase things in scientific literature, i.e., omitting "the work by" or "the paper by" and just saying "in Navid et al" but still it does not read well to me...]

Reply:
Perhaps also a comma after the citation?
" Dellar 2011, where..."?

Reply:
Hahaha, I agree with you completely.  It is standard and could imply we are discussing his organs.  I like "in the paper by".  Thanks for the suggestion and will change it right now and push it.